src/opts.js

```diff
@@ -1,16 +1,16 @@
 module.exports = (argv, env) => {
```

Comment:
This module was passed env but didn't use it. It should so removed process. everywhere